README.md

```diff
@@ -1 +1,2 @@
-# jupyterlab-zen-mode
\ No newline at end of file
+# JupyterLab Clarity Mode
```

Comment:
Why did you remove the hyphens?

Reply:
Very efficient use of PR team

Reply:
@markellekelly The hyphens originate from the repo name but I've changed it to a h1.